common/json.go

```diff
@@ -61,6 +67,18 @@ func (j *JSONHTTPClient) Get(ctx context.Context,
 	return jsonBody, nil
 }
 
+func (j *JSONHTTPClient) getURL(u string) (string, error) {
+	if strings.HasPrefix(u, "http://") || strings.HasPrefix(u, "https://") {
```

Comment:
aren't both of these conditions the same?

Reply:
one has an `s` in it